File: Source/C#/DevnologyFitnesseDojo/DevnologyFitnesseDojo/Fitnesse/Configuration/ConfigurationTest.cs
namespace DevnologyFitnesseDojo.Fitnesse.Configuration
{
    public class ConfigurationTest
    {
        private string value;

        public void SetInput(string value)
        {
            this.value = value;
        }

        public string Output()
        {
             return value; 
        }
    }
}

File: Source/Java/nl/devnology/fitnesse/configuration/ConfigurationTest.java
package nl.devnology.fitnesse.configuration;

/**
 * @author Erik Pragt
 */
public class ConfigurationTest {
    private String value;

    public void setInput(String input) {
        this.value = input;
    }

    public String output() {
        return value;
    }
}
